app/jobs/rainy_day_texter_job.rb

```diff
@@ -0,0 +1,30 @@
+class RainyDayTexterJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    users_with_phone_numbers.each do |user|
+      send_rainy_day_text(user)
+    end
+    RainyDayTexterJob.set(wait_until: early_next_morning).perform_later
```

Comment:
`early_next_morning` this is awesome
general shout-out for readability-of-code for this.